spec/rails_helper.rb

```diff
@@ -0,0 +1,86 @@
+# require database cleaner at the top level
+require 'database_cleaner'
+# This file is copied to spec/ when you run 'rails generate rspec:install'
+require 'spec_helper'
+ENV['RAILS_ENV'] ||= 'test'
+require File.expand_path('../../config/environment', __FILE__)
+# Prevent database truncation if the environment is production
+abort("The Rails environment is running in production mode!") if Rails.env.production?
```

Comment:
why is this for?

comment saying "Prevent database truncation" but it depends on configuration